src/app/containers/RadioPageBlocks/Blocks/Unknown/index.jsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 
 /* eslint-disable react/no-danger */
 const UnknownBlockContainer = () => (
+  // eslint-disable-next-line jam3/no-sanitizer-with-danger
   <div dangerouslySetInnerHTML={{ __html: `<!-- unknown block -->` }} />
```

Comment:
I think outputting an html comment here doesn't add that much value

Reply:
I've totally nuked this component in a separate PR. https://github.com/bbc/simorgh/pull/4169